src/libraries/System.Private.CoreLib/src/System/Math.cs

```diff
@@ -317,13 +317,31 @@ public static int DivRem(int a, int b, out int result)
             // Restore to using % and / when the JIT is able to eliminate one of the idivs.
             // In the meantime, a * and - is measurably faster than an extra /.
 
+            // TODO: implement DivRem intrinsic for Mono
+#if !MONO
+            if (X86Base.IsSupported)
+            {
+                (int quitient, result) = X86Base.DivRem((uint)a, a >> 31, b);
```

Comment:
typo "quotient" -- and below

Reply:
The variable name actually conflicts with outer one. Using q and r for those.